### PR TITLE
1.9.1prep

### PR DESCRIPTION
**Pull request by BGmot**
SUMMARY

New colellction version PR.
Fixes https://github.com/ansible-collections/community.zabbix/issues/869

ISSUE TYPE

New collection version Pull Request

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,24 @@ community.zabbix Release Notes
 .. contents:: Topics
 
 
+v1.9.1
+======
+
+Major Changes
```
Comment (D3DeFi) [inline CHANGELOG.rst:11]: minor, major is reserved for backwards incompatible i think

**Review by D3DeFi — APPROVED** [review]
Very quick prep :) Looks good, although I would change major -> minor first

File: CHANGELOG.rst (hunk 1 above)
Comment (BGmot) [reply CHANGELOG.rst:11]: do I change it via antsibul--changelog or directly  in CHANGELOG.rst?

Comment (D3DeFi) [reply CHANGELOG.rst:11]: I alywas updated either changelog/fragments/XYZ or changelog/changelog.yml and re-run antsibull-changelog

**Pull request merged**